### PR TITLE
Updated node, talib, tulind versions

### PR DESCRIPTION
**Pull request by mcsgroi**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Updated node, talib, tulind versions.

* **What is the current behavior?** (You can also link to an open issue here)
Doesn't launch node server currently.

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by mark-sch** [comment]
Hi, thanks, you updated talib, tulind and node to latest version, did you test to use these libs with node v10? This is the preferred, tested node version right now. Updating to the latest v15 node version will at least have the impact of heavy testing and I fear not all will run out of the box.

**Comment by mcsgroi** [comment]
Hello,
I wasn't sure what the preferred testing approach for this was. Everything
appeared to launch as expected although I'm new to this project. Figured
since I had the change locally I'd push it up in case you were interested
in merging it. I also won't be offended if you want to close this PR.

**Comment by XL-Reaper** [comment]
Hi, I tried this commit and some things break with Node 15 - for instance the exchange list stays empty.

**Review by XL-Reaper — CHANGES_REQUESTED** [review]
Forgot the changes - skip this comment :)

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:15
```
Comment (XL-Reaper) [inline Dockerfile:1]:
```suggestion
FROM node:10
```

File: package.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
     "sinon": "^4.2.0"
   },
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=15.2.1"
```
Comment (XL-Reaper) [inline package.json:67]:
```suggestion
    "node": ">=10.13.0"
```

**Review by XL-Reaper — CHANGES_REQUESTED** [review]
Change back to Node 10 so exchange selection doesn't break

**Pull request closed without merging**